### DISCUSSION TITLE
通过 JSON Schema 的形式有没有办法实现自定义 Form Item 中的 label 组件

### DISCUSSION THREAD

**Discussion by pandazki** [alibaba/formily#1270] [Q&A 2.x] [answered]
使用 JSON Schema 的形式 ReactNode 没有找到传递的方式

**Comment by janryWang** [comment, marked as answer]
用表达式，外部传入

**Reply by pandazki** [reply]
看了看源码~ 通过将 React 和 Lv1Label 组件放在 scope 中，再使用下面的方式指定，没有报错，在调试工具中也看到了对应的 ReactNode，但是实际渲染的 Label 还是原本的 title 文本
```
          "x-decorator-props": {
            "label": "{{React.createElement(Lv1Label, {}, null)}}",
          }
```
![image](https://user-images.githubusercontent.com/754493/115679466-329c1600-a385-11eb-91f9-5d8b42c7ccb4.png)

**Reply by janryWang** [reply]
传到title属性上去，label被title覆盖了
